### ISSUE TITLE
Incorrectly named drivers

### ISSUE THREAD

**Issue by saad-ali** [ceph/ceph-csi#222]
**Describe the bug**
Per the [CSI spec](https://github.com/container-storage-interface/spec/blob/master/spec.md#getplugininfo) the name of a CSI driver must conform to the following requirements:

```
The name MUST follow domain name notation format (https://tools.ietf.org/html/rfc1035#section-2.3.1).

It SHOULD include the plugin's host company name and the plugin name, to minimize the possibility of collisions.

It MUST be 63 characters or less, beginning and ending with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), dots (.), and alphanumerics between.

This field is REQUIRED.
```
The current names are:

* csi-cephfsplugin
* csi-rbdplugin

Consider renaming to include the fully qualified domain name, maybe something like:

* cephfsplugin.csi.ceph.com
* rbdplugin.csi.ceph.com

To be clear the sepc says `SHOULD include the plugin's host company name and the plugin name` so this is not a requirement. The existing names already include ceph and rbd so the chance of collision is low, but for the sake of consistency and lowering the chance of collision further I recommend the rename.

**Kubernetes and Ceph CSI Versions**
Kubernetes: Kubernetes 1.9 - 1.13
Ceph CSI: 0.3.0 and 1.0.0

**Ceph CSI Driver logs**
N/A

**To Reproduce**
N/A

**Expected behavior**
N/A

**Comment by humblec** [comment]
@saad-ali sure. thanks for the report. We are addressing it via pr#223.

**Comment by dungdm93** [comment]
Hello guys,

My suggestion the better names are:
* `csi.ceph.com/rbd`
* `csi.ceph.com/cephfs`

Because it follow the same pattern as in-tree Kubernetes plugins as well as Rook:
* `kubernetes.io/rbd`
* `kubernetes.io/gce-pd`
* `kubernetes.io/aws-ebs`
* `ceph.rook.io/block`
* ...

**Comment by Madhu-1** [comment]
@rootfs @gman0 @humblec @ShyamsundarR can you guys  suggest which driver naming convention we have to follow?

**Comment by humblec** [comment]
@Madhu-1 
```csi.ceph.com/rbd``` or ```rbd.csi.ceph.com .
I would vote for `csi.ceph.com/rbd` , but lets hear from others too.

**Comment by Madhu-1** [comment]
we are not doing much for the fixes in the master branch. do we still need to do fixes in both master and V1.0 branch?

**Comment by JohnStrunk** [comment]
Per my reading of the spec, "/" is not a permitted character.

**Comment by ShyamsundarR** [comment]
Based on comments above I would suggest that we use the following defaults,
`rbd.csi.ceph.com` and `fs.csi.ceph.com` (dropping the additional ceph reference in the other option suggested `cephfs.csi...`)

**Comment by humblec** [comment]
@JohnStrunk Thanks for the catch.. !! yeah, `/` cant be in driver name. So the better choice here would be 
For RBD: `rbd.csi.ceph.com`
For CephFS: `fs.csi.ceph.com` or `cephfs.csi.ceph.com`?  the latter looks good to me.
@Madhu-1 @rootfs @JohnStrunk @gman0 @travisn any preference?

**Comment by dungdm93** [comment]
+1 for `cephfs.csi.ceph.com`

**Comment by travisn** [comment]
agreed on `cephfs.csi.ceph.com` and `rbd.csi.ceph.com`

**Comment by Madhu-1** [comment]
@gman0 am reopening till we get  same changes in the master branch

**Comment by humblec** [comment]
As an update , the changes in `v1.0` branch is done and the patch is merged!

**Comment by Madhu-1** [comment]
Fixed in #262
